the-collab-lab/tcl-62-smart-shopping-list
# Chakra setup

## Description

This code sets up Chakra UI as the CSS library that will be used to style this app. It extends Chakra's default theme with custom colors and fonts. It also applies some Chakra features to ensure that the setup is working correctly.

## Acceptance Criteria

Chakra UI components should be able to be successfully imported in each React component.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|     | :scroll: Docs              |
| ✓   | :sparkling_heart: Styling     |

### Before
![Screenshot 2023-09-28 at 4 35 45 PM](https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/115492619/9043921e-ff22-48e1-a641-8802c6eab07b)

### After
![Screenshot 2023-09-28 at 4 34 39 PM](https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/115492619/97bd4131-1718-4164-b62d-6f4f7b30d733)

## Testing Steps / QA Criteria

1. Switch to the `chakra-setup` branch on your local computer
2. Run `npm i` to install the newest dependencies
3. Run `npm start` to start the application
4. In your web browser ensure that the app features are still working. Verify that some custom colors and fonts are visible as per the agreed upon style guide.
5. Clear tokens from local storage and check that features of home page and working and verify that some styling is visible.
6. If you create new lists, be sure to delete them from Firestore to keep the database easy to navigate.
